app/models/announcement/block/donation_summary.rb

```diff
@@ -51,7 +51,7 @@ def start_date_param
         if self.parameters["start_date"].present?
           DateTime.parse(self.parameters["start_date"])
         else
-          self.parameters["start_date"] ||= 1.month.ago
+          self.parameters["start_date"] ||= 1.month.ago.to_s
```

Comment:
```suggestion
          self.parameters["start_date"] ||= 1.month.ago.to_s
          1.month.ago
```
The `start_date_param` method should still return a `DateTime` to be consistent with the non-nils

Reply:
good catch yes